local-cli/server/findSymlinksPaths.js

```diff
@@ -6,7 +6,28 @@ module.exports = function findSymlinksPaths(lookupFolder) {
   const folders = fs.readdirSync(lookupFolder);
   const resolvedSymlinks = folders.map(folder => path.resolve(lookupFolder, folder))
     .filter(folderPath => fs.lstatSync(folderPath).isSymbolicLink())
-    .map(symlink => path.resolve(process.cwd(), fs.readlinkSync(symlink)));
+    .map(symlink => {
+      const symlinkCache = new Set();
+      let prevSymlink;
+      let nextSymlink = symlink;
+      while (true) {
```

Comment:
Seems redundant, no? We already do the `filter` call to make sure only symlinks are present.